src/emu/arm/src/arm_dynarmic.cpp

```diff
@@ -339,7 +340,8 @@ namespace eka2l1::arm {
 
         std::uint64_t GetTicksRemaining() override {
             return static_cast<std::uint64_t>(common::max<std::int64_t>(static_cast<std::int64_t>(parent.ticks_target)
-                - parent.ticks_executed, 0));
+                    - parent.ticks_executed,
+                0));
```

Comment:
Please adjust this format change

Reply:
Done 25695f6c3a341d114a4a05d2810742d330903450

Reply:
Done bd2b309